apps/admin-portal/src/components/identity-providers/forms/components/common-pluggable-component-form.tsx

```diff
@@ -0,0 +1,198 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { Button, Grid } from "semantic-ui-react";
+import { CommonConstants, FieldType, getFieldType, getPropertyField } from "../helpers";
+import {
+    CommonPluggableComponentFormPropsInterface,
+    CommonPluggableComponentInterface,
+    CommonPluggableComponentMetaPropertyInterface,
+    CommonPluggableComponentPropertyInterface
+} from "../../../../models";
+import { Forms, FormValue } from "@wso2is/forms";
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { getPropertyMetadata } from "../../utils";
+
+
```

Comment:
I additional line

Reply:
Fixed in c0e366925209b7d80078ad6edc9077bb13ca9b5b